app/public/src/pages/game.tsx

```diff
@@ -127,47 +124,52 @@ export default function Game() {
 
   const MAX_ATTEMPS_RECONNECT = 3
 
-  function connectToGame(attempts = 1) {
-    logger.debug(`connectToGame attempt ${attempts} / ${MAX_ATTEMPS_RECONNECT}`)
-    const cachedReconnectionToken = localStorage.getItem(
-      "cachedReconnectionToken"
-    )
-    if (cachedReconnectionToken) {
-      connecting.current = true
-      const statusMessage = document.querySelector("#status-message")
-      if (statusMessage) {
-        statusMessage.textContent = `Connecting to game...`
-      }
+  const connectToGame = useCallback(
```

Comment:
diff looks scary here, but only changes to this function were to convert to `useCallback` and adding async/await on line 155